### ISSUE TITLE
it doesn't open streams 

### ISSUE THREAD

**Issue by ChaviHachav** [Aayush9029/NativeTwitch#39]
### when clicking on play using IINA it just opens IINA

Logs:
 ```
B13622AB-7606-42FB-A443-27240F3FBE55 : 2023-04-30 19:55:54 +0000 >> Loading Streams

 E5BE4A22-2F84-484E-AA25-0E564236CF9F : 2023-04-30 19:55:55 +0000 >> Streams Loaded

 22998F0D-EADD-4C0B-8B83-717BAA957C85 : 2023-04-30 19:55:55 +0000 >> {
  "data" : [
    {
      "user_id" : "207813352",
      "is_mature" : false,
      "tag_ids" : [

      ],
      "type" : "live",
      "thumbnail_url" : "https:\/\/static-cdn.jtvnw.net\/previews-ttv\/live_user_hasanabi-{width}x{height}.jpg",
      "started_at" : "2023-04-30T18:02:45Z",
      "viewer_count" : 29942,
      "title" : "100T VOGUE VALKYRAE JOINS ME FOR SUNDAY PUPDAY + MURDER + POSSIBLE SCARY GAMING",
      "id" : "42141597019",
      "language" : "en",
      "game_name" : "Just Chatting",
      "game_id" : "509658",
      "tags" : [
        "news",
        "politics",
        "adhd",
        "English"
      ],
      "user_name" : "HasanAbi",
      "user_login" : "hasanabi"
    },
    {
      "user_id" : "3389768",
      "is_mature" : true,
      "tag_ids" : [

      ],
      "type" : "live",
      "thumbnail_url" : "https:\/\/static-cdn.jtvnw.net\/previews-ttv\/live_user_philza-{width}x{height}.jpg",
      "started_at" : "2023-04-30T19:28:12Z",
      "viewer_count" : 27613,
      "title" : "QSMP - Goodbye, my little hero",
      "id" : "46828179644",
      "language" : "en",
      "game_name" : "Minecraft",
      "game_id" : "27471",
      "tags" : [
        "Philza",
        "muchfunny",
        "blockgame",
        "distracted",
        "English"
      ],
      "user_name" : "Philza",
      "user_login" : "philza"
    },
    {
      "user_id" : "247808909",
      "is_mature" : false,
      "tag_ids" : [

      ],
      "type" : "live",
      "thumbnail_url" : "https:\/\/static-cdn.jtvnw.net\/previews-ttv\/live_user_qtcinderella-{width}x{height}.jpg",
      "started_at" : "2023-04-30T19:18:36Z",
      "viewer_count" : 2754,
      "title" : "Baking Kirby cake  | !bakealong !wineaboutit | !youtube !react",
      "id" : "42141794091",
      "language" : "en",
      "game_name" : "Just Chatting",
      "game_id" : "509658",
      "tags" : [
        "English"
      ],
      "user_name" : "QTCinderella",
      "user_login" : "qtcinderella"
    },
    {
      "user_id" : "129180414",
      "is_mature" : false,
      "tag_ids" : [

      ],
      "type" : "live",
      "thumbnail_url" : "https:\/\/static-cdn.jtvnw.net\/previews-ttv\/live_user_realfitmc-{width}x{height}.jpg",
      "started_at" : "2023-04-30T19:27:20Z",
      "viewer_count" : 2336,
      "title" : "QSMP - Day 40 - Goodbye Ramon (Wig Stream)",
      "id" : "41297481833",
      "language" : "en",
      "game_name" : "Minecraft",
      "game_id" : "27471",
      "tags" : [
        "English",
        "minecraft",
        "qsmp"
      ],
      "user_name" : "RealFitMC",
      "user_login" : "realfitmc"
    },
    {
      "user_id" : "24713999",
      "is_mature" : false,
      "tag_ids" : [

      ],
      "type" : "live",
      "thumbnail_url" : "https:\/\/static-cdn.jtvnw.net\/previews-ttv\/live_user_smajor-{width}x{height}.jpg",
      "started_at" : "2023-04-30T18:10:05Z",
      "viewer_count" : 1291,
      "title" : "Void Rising Event! Come watch! :D",
      "id" : "46827932556",
      "language" : "en",
      "game_name" : "Minecraft",
      "game_id" : "27471",
      "tags" : [
        "lgbtqia",
        "gay",
        "scotland"
      ],
      "user_name" : "Smajor",
      "user_login" : "smajor"
    },
    {
      "user_id" : "47764708",
      "is_mature" : false,
      "tag_ids" : [

      ],
      "type" : "live",
      "thumbnail_url" : "https:\/\/static-cdn.jtvnw.net\/previews-ttv\/live_user_slimecicle-{width}x{height}.jpg",
      "started_at" : "2023-04-30T19:54:56Z",
      "viewer_count" : 0,
      "title" : "QSMP: The Egg Finale",
      "id" : "48405252365",
      "language" : "en",
      "game_name" : "Minecraft",
      "game_id" : "27471",
      "tags" : [
        "Comedy",
        "minecraft",
        "English"
      ],
      "user_name" : "Slimecicle",
      "user_login" : "slimecicle"
    }
  ],
  "pagination" : {

  }
}

 AD944C30-ABBD-4C03-AA1D-81EFE0567ADD : 2023-04-30 19:55:56 +0000 >> :🎉 Success 🎉

```

**Comment by Aayush9029** [comment]
It seems to be Youtube's issue which is mentioned [here](https://github.com/iina/iina/issues/4223), you could try using yt-dlp backend.

https://github.com/iina/iina/issues/4223


---

Since there’s still no easy comment that summarizes it all, these are the steps for the workaround

1. `brew install yt-dlp`
2. `which yt-dlp` and remove the `/yt-dlp` in the end
  _(in my case it was `/usr/local/bin/yt-dlp` => `/usr/local/bin`)_
3. In `IINA -> Settings -> Network -> youtube-dl` add the folder from above (e.g. `/usr/local/bin`)
![image](https://user-images.githubusercontent.com/1133858/220641017-95be77ac-27c0-4f53-9a27-5ea9c6aadeed.png)
4. Restart IINA


_Originally posted by @FlorianWendelborn in https://github.com/iina/iina/issues/4223#issuecomment-1440057037_

**Comment by ChaviHachav** [comment]
I'm using native twitch

**Comment by Aayush9029** [comment]
Seems to work fine for me, I tried using iina as well as QuickTime.

Make sure to specify streamlink path in settings.

<img width="1512" alt="CleanShot 2023-05-01 at 16 06 44@2x" src="https://user-images.githubusercontent.com/43297314/235522028-8acf8021-05a5-4c0b-9e61-88eb27bc8dc6.png">


> I'm using native twitch


Right, I got my native apps mixed up 😅
Anyways https://github.com/Aayush9029/NativeTwitch/tree/v3 should fix this issue if adding streamlink path doesn't work. It is in development phase however.
